extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -541,39 +592,85 @@ public void addRoute(RuntimeValue<Router> router, Function<Router, Route> route,
 
     private static class WebDeploymentVerticle extends AbstractVerticle {
 
-        private final int port;
-        private final int httpsPort;
-        private final String host;
```

Comment:
This variables are redundant since their values are in `httpOptions` and `httpsOptions`.